index.js

```diff
@@ -70,13 +85,16 @@ function run(configPath, options, callback) {
         ));
     }
 
-    var maxRetries = parseInt(options.maxRetries, 10) || 0,
-        maxConcurrentWorkers = parseInt(options.maxConcurrentWorkers, 10)
-            || require('os').cpus().length,
-        workers = workerFarm({
-            maxRetries: maxRetries,
-            maxConcurrentWorkers: maxConcurrentWorkers
-        }, require.resolve('./src/webpackWorker'));
+    if (!validate(options)) {
+        return Promise.reject(new Error(
+          'Options validation failed:\n' +
+          validate.errors.map(function(error) {
+            return 'Property: "options' + error.dataPath + '" ' + error.message;
+          }).join('\n')
+        ));
+    }
+
+    var workers = workerFarm(options, require.resolve('./src/webpackWorker'));
```

Comment:
That'd be perfect. Would help to avoid accidental breaking if at some point worker-farm adds a new option that we're also interpreting.

Thank you so much for doing this!